packages/integration-sdk-cli/src/neo4j/neo4jUtilities.ts

```diff
@@ -50,3 +55,30 @@ export function buildPropertyParameters(propList: Object) {
 
   return propertyParameters;
 }
+
+// Start and end type helper functions.  Prepends a : to any nonempty results for
+// immediate use in a Neo4j command.
+export function getFromType(relationship: Relationship): String {
```

Comment:
Unit tests?

Reply:
Good call out!  I'll plan to add and see about getting another approval.   Thanks!

Reply:
Added in one of the latest commits.